### PR TITLE
JW-892: Port KSync test

### PR DESCRIPTION
**Pull request by sodar**

### CODE REVIEW

**Comment by katrybacka** [comment]
Janie, przetestuj proszę

**Review by katrybacka — APPROVED** [review]

File: rules.py [hunk 1]
```diff
@@ -234,11 +234,14 @@ def UnitTest(env, name, sources, **kwargs):
...
-    if sys.platform != 'darwin' and env.get('OPT') != 'coverage' and \
-           not env['ENV'].has_key('NO_HEAPCHECK') and \
-           env.get('OPT') != 'valgrind':
+    if sys.platform != 'darwin' and sys.platform != 'win32' and env.get('OPT') != 'coverage' and \
+           not env['ENV'].has_key('NO_HEAPCHECK') and env.get('OPT') != 'valgrind':
         test_env.Append(LIBPATH = '#/build/lib')
         test_env.Append(LIBS = ['tcmalloc'])
+
+    if sys.platform == 'win32':
+        test_env.Append(LIBS = ['Iphlpapi'])
```
Comment (mclapinski) [inline rules.py]: I don't think this belongs here.

**Review by mclapinski — CHANGES_REQUESTED** [review]

**Comment by mclapinski** [comment]
Please remember that after the merge you have to rebase this changes onto windows4.0 and tell me to add them to PR on Gerrit

**Comment by jacek-izykowski** [comment]
Janie, przetestuj, proszę.

File: rules.py [hunk 2]
```diff
@@ -234,11 +234,11 @@ def UnitTest(env, name, sources, **kwargs):
...
 
     # Do not link with tcmalloc when running under valgrind/coverage env.
-    if sys.platform != 'darwin' and env.get('OPT') != 'coverage' and \
-           not env['ENV'].has_key('NO_HEAPCHECK') and \
-           env.get('OPT') != 'valgrind':
+    if sys.platform != 'darwin' and sys.platform != 'win32' and env.get('OPT') != 'coverage' and \
+           not env['ENV'].has_key('NO_HEAPCHECK') and env.get('OPT') != 'valgrind':
         test_env.Append(LIBPATH = '#/build/lib')
         test_env.Append(LIBS = ['tcmalloc'])
+
```
Comment (mclapinski) [inline rules.py]: useless

**Review by mclapinski — CHANGES_REQUESTED** [review]

File: rules.py (hunk 2 above)
Comment (jacek-izykowski) [reply rules.py]: This is valid change that improves readability.

**Review by mclapinski — APPROVED** [review]

**Pull request merged**